labseu2-workout-tracker/client
# Modify authentication component

##  What does this PR do
Modifies Login and Signup to use Ant Design and provide error message feedback

##  Description of tasks to be completed
add authenticationActions for use by redux
add authentication reducers for use by redux
add auth function to verify token
Add Private route to protect restricted routes
Remove stateful logic to use redux store
add authentication to combine reducers
modify login component to use ant design
modify login component to use ant design
modify signup component to use ant design
change class to className
add logo as favicon and change app title
fix merge conflicts issues

##  How should this manually be tested
checkout the branch `modify-authentication-component` 
run `yarn install`
start the server and check out the login and signup components

##  What are the relevant Trello board stories
(https://trello.com/c/UPuD8XKJ/143-ui-for-login-and-signup-errors)[UI for Login and Signup Errors]

##  Screenshots (If appropriate)
<img width="808" alt="Screenshot 2019-09-06 at 9 17 54 AM" src="https://user-images.githubusercontent.com/12853775/64412483-5b27ad00-d087-11e9-8863-3cffc394db26.png">
<img width="454" alt="Screenshot 2019-09-06 at 9 18 06 AM" src="https://user-images.githubusercontent.com/12853775/64412485-5bc04380-d087-11e9-8092-dfe8620cb5f0.png">
<img width="849" alt="Screenshot 2019-09-06 at 9 18 28 AM" src="https://user-images.githubusercontent.com/12853775/64412486-5bc04380-d087-11e9-9be6-72c026d0c4ff.png">


##  Questions (If any):
